public/stylesheet.css

```diff
@@ -376,6 +376,9 @@ body.no-js .control-field__title {
   background-color: transparent;
   color: var(--color-text-default);
   height: 2.4rem;
+}
+
+.control__button__padding {
```

Comment:
Why not add this padding so the buttons being added here look (arguably) better?

| Current | With padding |
| :---: | :---: |
| ![current](https://user-images.githubusercontent.com/3742559/139751665-fe3b4039-5859-4fb6-ba06-dbc937d10485.png) | ![with padding](https://user-images.githubusercontent.com/3742559/139751716-b508f9f0-c969-4e6c-8572-d5b25232c421.png) |

I know we've been going back and forth about this, any objections to this suggestion?